testCommon/src/testFixtures/java/com/virtuslab/gitmachete/testcommon/TestProcessUtils.java

```diff
@@ -22,18 +23,17 @@ public static String runProcessAndReturnStdout(Path workingDirectory, int timeou
     boolean completed = process.waitFor(timeoutSeconds, TimeUnit.SECONDS);
 
     String stdout = IOUtils.toString(process.getInputStream(), StandardCharsets.UTF_8);
-
+    String stderr = IOUtils.toString(process.getErrorStream(), StandardCharsets.UTF_8);
     String commandRepr = Arrays.toString(command);
-    if (!completed || process.exitValue() != 0) {
-      System.out.println("Stdout of ${commandRepr}: ${System.lineSeparator()}");
-      System.out.println(stdout);
-      System.err.println("Stderr of ${commandRepr}: ${System.lineSeparator()");
-      System.err.println(IOUtils.toString(process.getErrorStream(), StandardCharsets.UTF_8));
-    }
-
-    Assert.assertTrue("command ${commandRepr} has not completed within ${timeoutSeconds} seconds;", completed);
+    val stdoutMessage = "Stdout of ${commandRepr}: ${System.lineSeparator()}";
```

Comment:
i think we need to have one condition that will check when `process.exitValue !=0`

![image](https://user-images.githubusercontent.com/45738614/196506347-c5fb388d-96db-4e83-80e6-601aed395f47.png)

Reply:
> one condition that will check when process.exitValue !=0

That's exactly what you've got here... the problem is that string interpolations don't work for some reason ☹️ pls use `+`s everywhere then so that we can see the stdout/stderr

Reply:
i have not used `+s` and now this is the error i am getting 

![image](https://user-images.githubusercontent.com/45738614/196614539-3a7f4e30-cc87-4052-bcf1-05a9e2d0abba.png)

Reply:
Wow okay that's much clearer now :) okay... consider installing Windows Subsystem for Linux then :)